src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -93,11 +94,26 @@
    */
   public final String viewIdResourceName;
 
+  /**
+   * Is view an instance of WebView.
+   */
+  private final boolean isWebView;
```

Comment:
I feel like this property should be a calculated property, as it is not part of Android.View. We do make exceptions, but I'm not sure this should be one?